OrbitLinuxTracing/TracerThread.cpp

```diff
@@ -301,20 +291,34 @@ void TracerThread::ProcessSampleEvent(const perf_event_header& header,
   bool is_uretprobe = is_probe && (header.size == size_of_uretprobe);
   bool is_uprobe = is_probe && !is_uretprobe;
 
+  pid_t pid;
+  if (is_uretprobe) {
+    pid = ReadUretprobesRecordPid(ring_buffer);
+  } else {
+    pid = ReadSampleRecordPid(ring_buffer);
+  }
+
+  if (pid != pid_) {
+    ring_buffer->SkipRecord(header);
+    return;
+  }
+
   if (is_uprobe) {
     auto event =
         ConsumeSamplePerfEvent<UprobesWithStackPerfEvent>(ring_buffer, header);
     event->SetFunction(uprobes_fds_to_function_.at(fd));
     event->SetOriginFileDescriptor(fd);
     DeferEvent(std::move(event));
     ++stats_.uprobes_count;
+
```

Comment:
Nit: Delete empty line at end of block.

Reply:
In the absence of a written code-style for ORBIT, I suggest we follow standard Google code style, in which case these lines have to be deleted.

Reply:
The Google C++ style guide says
"Blank lines inside a chain of if-else blocks may well help readability."

Reply:
Ok, I wasn't aware. Then it's a matter of opinion. Keep them if you want.